code/script.js

```diff
@@ -2,18 +2,116 @@
 
 // Step 1 - Welcome and introduction
 // Your code goes here
+// Step 1a
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
+// Step 1b A prompt asking for the clients name
+const name = prompt('What is your name?')
+// Step 1c An alert confirmin the clients name
+alert(`Ciao ${name.toUpperCase()}, Let's get started with your order!`)
 
 // Step 2 - Food choice
-// Your code goes here
+
+// Prompt with a meal selection
+const mealSelection = prompt(`Please, select what dish you would have us cook for you:
+
+  Press: 1 - for Pizza
+  Press: 2 - for Pasta
+  Press: 3 - for Salad`)
+
+
+if (mealSelection === '1') {
+  alert('You have selected a pizza!')
+} else if (mealSelection === '2') {
+  alert('You have selected pasta!')
+} else if (mealSelection === '3') {
+  alert('You have selected a salad!')
+} else { 
+  alert('Invalid choice. Please use a number between 1 and 3')
+}
```

Comment:
Just for fun, I tried entering an invalid choice, and it had me go ahead and move on to the next question anyway. There are a couple ways to make this more logical: For example, I used a while loop to force the user to enter a valid number or else I would continuously prompt the user to enter 1 2 or 3. Another option is to use switch statements where the default case (anything other than options 1, 2, or 3) can alert an invalid choice and you can use location.reload() which should reload the page to start over.